src/sanitize-argument.ts

```diff
@@ -55,6 +56,20 @@ export async function sanitize_all(argv: any) {
     throw err;
   });
 
+  // sanitize Custom Main Page
+  if (argv.customMainPage) {
+    await axios.get(`${mwUrl}/wiki/${argv.customMainPage.replace(/ /g,'_')}`)
```

Comment:
* we should keep the space replacement
* you shoudl not hardcode the path `wiki`, you should use whatever hase been given in Mediawiki.